docs/handlers.md

```diff
@@ -1,52 +1,51 @@
 ??? example "All supported formats"
     | Format        | Type                                 | Fully supported?    |
     | :------------ | :----------------------------------- | :-----------------: |
+    | [`7-ZIP`](#7-zip) | ARCHIVE | :octicons-check-16: |
     | [`AR`](#ar) | ARCHIVE | :octicons-check-16: |
     | [`ARC`](#arc) | ARCHIVE | :octicons-check-16: |
     | [`ARJ`](#arj) | ARCHIVE | :octicons-check-16: |
-    | [`HP BDL`](#hp-bdl) | ARCHIVE | :octicons-check-16: |
-    | [`INSTAR BNEG`](#instar-bneg) | ARCHIVE | :octicons-check-16: |
-    | [`BZIP2`](#bzip2) | COMPRESSION | :octicons-check-16: |
+    | [`ANDROID EROFS`](#android-erofs) | FILESYSTEM | :octicons-check-16: |
+    | [`ANDROID SPARSE`](#android-sparse) | FILESYSTEM | :octicons-check-16: |
+    | [`AUTEL ECC`](#autel-ecc) | ARCHIVE | :octicons-check-16: |
     | [`CAB`](#cab) | ARCHIVE | :octicons-check-16: |
```

Comment:
BZIP2 moved, and also there are other changes later, that would not appear if sorting would be stable.

Reply:
I decided to change sorting from the handler name to the handler's doc name, it provides a better representation. That's why it moved around.

Fixed the code to make ruff happy and added pymdown-extensions to `package.nix` to fix the Nix build.